tools/azure-pipelines/cache_docker_images.sh

```diff
@@ -28,7 +28,7 @@ then
 fi
 
 # This is the pattern that determines which containers we save.
-DOCKER_IMAGE_CACHE_PATTERN="testcontainers|kafka|postgres|mysql|pulsar|cassandra"
+DOCKER_IMAGE_CACHE_PATTERN="testcontainers|kafka|postgres|mysql|pulsar|cassandra|schema_registry"
```

Comment:
you sure this doesn't need a `-` instead of `_`?

Reply:
FFS. Fixed